modules/UpgradeWizard/language/en_us.lang.php

```diff
@@ -177,7 +177,7 @@
     'LBL_UW_SKIPPED_FILES_TITLE' => 'Skipped Files',
     'LBL_UW_SQL_RUN' => 'Check when SQL has been manually run',
     'LBL_UW_START_DESC' => 'This wizard will assist you in upgrading this SuiteCRM instance.',
-    'LBL_UW_START_DESC2' => 'Note: We recommend that you backup the SuiteCRM database and the system files (all of the files in the SuiteCRM folder) before upgrading your production system. We highly recommend that you perform a test upgrade on a cloned instance of your production system first. You may also need to run "composer update" if you have customised the composer.json file',
+    'LBL_UW_START_DESC2' => 'Note: We highly recommend that you create a copy of the SuiteCRM instance you use in production, and test the upgrade package before deploying the new version to production. After the upgrade process has completed, please run <pre>composer update</pre> to ensure that you all the latest.',
```

Comment:
the end of the sentence needs a revision: see "to ensure that you all the latest."